documentation/doc_sncf/source/integration.rst

```diff
@@ -466,21 +460,12 @@ the selected public transport object or coordinates.
 
 There are two ways to access this api.
 
-The first one is: `<https://api.navitia.io/v1/{a_path_to_resource}/journeys>`_ it will retrieve 
-all the journeys from the resource (*isochrones*).
-
 The other one, the most used, is to access the 'journey' api endpoint: `<https://api.navitia.io/v1/journeys?from={resource_id_1}&to={resource_id_2}&datetime={datetime}>`_ .
 
 +-------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | *Note*                                                                                                                                                      |
 |                                                                                                                                                             |
-| Navitia.io handle lot's of different data sets (regions). Some of them can overlap. For example opendata data sets can overlap with private data sets.      |
-|                                                                                                                                                             |
-| When using the journeys endpoint the data set used to compute the journey is chosen using the possible datasets of the origin and the destination.          |
-|                                                                                                                                                             |
-| For the moment it is not yet possible to compute journeys on different data sets, but it will one day be possible (with a cross-data-set system).           |
-|                                                                                                                                                             |
-| If you want to use a specific data set, use the journey api within the data set: `<https://api.navitia.io/v1/coverage/{your_dataset}/journeys>`_            |
+| The API SNCF handle to compute journey from "Stop Area" to " Stop Area" or "Stop Area" to "Administrative Region"            |
```

Comment:
I'm not sure, but I think it's more english to say:

"The SNCF API handle journey computation from "

As for the content, I think what you mean is that you cannot do coordinates to coordinates computation no ?